Documentation/release-notes/4375.md

```diff
@@ -0,0 +1,16 @@
+### AndroidExplicitCrunch feature removed
+
+In previous Xamarin.Android versions, setting the
+`AndroidExplicitCrunch` MSBuild property to `true` would configure the
+build to [crunch][crunch-upstream] PNG drawable resources using a
+separate MSBuild task before passing them to AAPT.
+
+This feature was never compatible with projects configured to use
+AAPT2 and was automatically disabled in those projects.  For projects
+configured to use the older AAPT tool, this feature could cause builds
+to rerun AAPT on every build, which may or may not improve build times
+depending on the project.  Due to these limitations, this feature has
```

Comment:
```suggestion
to rerun AAPT on every build.  Due to these limitations, this feature has
```

I think "improve build times" reads as backwards in the current context. One option could be something like: "which could hurt overall build times." But after rereading this, I think an even simpler option could be to leave off the last part of the sentence altogether because "rerun ... on every build" already suggests that the behavior was undesirable.